Add QMap include to fix building under Qt 5.13

This combines several patches which all add the same missing include:
- e13ab06f65a4e8b6259bccda1401abb9c1c9fdca
- 546ede9ef712bb1b156a55ce30df81c3203dde61
- 08fb06b63a78888e6703de6ac1a4fd6a0e2791eb
- bef6784596bf0b41287ebad81f138e1c8c2a6be2

Include is no longer added transitively.
---

--- a/plugins/betternotify/notify.h
+++ b/plugins/betternotify/notify.h
@@ -23,6 +23,7 @@
 #ifndef NOTIFY_H
 #define NOTIFY_H
 
+#include <QMap>
 #include <QQueue>
 #include <QPoint>
 #include <QPointer>
--- a/plugins/untiny/untiny.h
+++ b/plugins/untiny/untiny.h
@@ -27,6 +27,7 @@
 
 #include "plugin.h"
 
+#include <QMap>
 #include <QQueue>
 #include <QUrl>
 #include <QPointer>
--- a/plugins/uploaders/flickr/flickr.h
+++ b/plugins/uploaders/flickr/flickr.h
@@ -26,6 +26,8 @@
 
 #include "uploader.h"
 
+#include <QMap>
+
 /**
 @author Andrey Esin \<gmlastik@gmail.com\>
 */
diff --git a/plugins/uploaders/imageshack/imageshack.h b/plugins/uploaders/imageshack/imageshack.h
index d9ddc88..63d0e3f 100644
--- a/plugins/uploaders/imageshack/imageshack.h
+++ b/plugins/uploaders/imageshack/imageshack.h
@@ -26,6 +26,8 @@
 
 #include "uploader.h"
 
+#include <QMap>
+
 /**
 @author Mehrdad Momeny \<mehrdad.momeny@gmail.com\>
 */
--- a/plugins/uploaders/mobypicture/mobypicture.h
+++ b/plugins/uploaders/mobypicture/mobypicture.h
@@ -26,6 +26,8 @@
 
 #include "uploader.h"
 
+#include <QMap>
+
 class KJob;
 
 class Mobypicture : public Choqok::Uploader
--- a/plugins/uploaders/posterous/posterous.h
+++ b/plugins/uploaders/posterous/posterous.h
@@ -26,6 +26,8 @@
 
 #include "uploader.h"
 
+#include <QMap>
+
 class KJob;
 
 class Posterous : public Choqok::Uploader
--- a/plugins/uploaders/twitgoo/twitgoo.h
+++ b/plugins/uploaders/twitgoo/twitgoo.h
@@ -26,6 +26,8 @@
 
 #include "uploader.h"
 
+#include <QMap>
+
 class KJob;
 
 class Twitgoo : public Choqok::Uploader
--
